README.md

```diff
@@ -63,10 +48,6 @@ An opinioned coding guideline for React + Redux project @WORD UP
     * for example: `//TODO: (From Mark) hookup Ajax to rest server`
   
 ## React
-#### Add `propTypes` to every components
-  * It helps others to use the component easily and also help to catch some errors quickly.
```

Comment:
propTypes should have check by eslint

Reply:
Though Eslint helps us to catch it, we still should write it down into our guideline. So Eslint is a tool to check if the code applies to our rules.

Reply:
If we want to include `eslint` rule in style guild, this guideline will be at least 10 times bigger, than no one will want to read/maintain it.
That's why I hope to keep this as simple as possible.

Reply:
This is topically how other projects like `robucop` and `eslint-airbnb` does. They have a lint config and also a doc to explain what's the reason behind the eslint rule.

But, I do like the idea of including `.eslintrc` file into this repo. Then how about this: We include the `.eslintrc` into this repo. But if we change the rule that differents from the default(airbnb style for now), then we explain the change in this repo. If the rule is just following the default setting, then we don't need to write it down here.

Reply:
totally agree.

Reply:
this is a good idea 👏